benchmarking/nursery/benchmark_multiobjective/run_experiment.py

```diff
@@ -0,0 +1,117 @@
+from typing import Callable
```

Comment:
OK, this is a V0, but why cannot we use the generic code that is already there?

The only difference I see is that there are >1 `metric` and `mode`. We could extend the generic case to deal with that.

Reply:
I agree. The plan is to make this v0 work while being explicit (spelling out all the operations) to see what needs to be done for MO benchmarks. Then v1 will refactor this code to use `commons` while making any changes to commons to make it possible.

Reply:
Updating PR to re-use existing logic in commons